pkg/prometheus/promcfg_test.go

```diff
@@ -7231,11 +7231,13 @@ func TestScrapeConfigSpecConfigWithDockerSDConfig(t *testing.T) {
 						Port:               ptr.To(9100),
 						RefreshInterval:    ptr.To(monitoringv1.Duration("30s")),
 						HostNetworkingHost: ptr.To("localhost"),
-						Filters: &[]monitoringv1alpha1.DockerFilter{
+						Filters: []monitoringv1alpha1.Filter{
 							{Name: "dummy_label_1",
 								Values: []string{"dummy_value_1"}},
 							{Name: "dummy_label_2",
 								Values: []string{"dummy_value_2", "dummy_value_3"}},
+							{Name: "a_dummy_label_1",
+								Values: []string{"dummy_value_2", "dummy_value_3"}},
```

Comment:
I see that you're adding this "alphabetic out-of-order" filter here with the goal to test for determinism, right?

Should we run this test like 50x times, just to guarantee the determinism? Or running just once is enough?

Reply:
🤔 I didn't think of it that way tbh. But since the sorting algo is from the standard lib, we can rest easy? I'm not sure tbh. cc: @simonpasquier

Reply:
running once is enough.

Reply:
Sounds good to me :)